### PR TITLE
Real to complex

### PR DESCRIPTION
**Pull request by AdamNorberg**
Lathreas wanted a cast.

### CODE REVIEW

File: engine/Real.cs [hunk 1]
```diff
@@ -302,9 +304,16 @@ public Real(Real r)
...
 		public static explicit operator ulong(Real real) => (ulong) real._v;
 
+		/// <summary>
+		/// Convert a Real to a Complex with zero imaginary part.
+		/// </summary>
+		/// <param name="real">Real to convert.</param>
+		/// <returns>Complex with real part equal to the provided Real (as double) and zero imaginary part.</returns>
+		public static explicit operator Complex(Real real) => new Complex((double) real._v, 0.0);
+
 		#endregion
```
Comment (Lathreas) [inline engine/Real.cs:314]:
```suggestion
#endregion
```

File: engine/Real.cs [hunk 2]
```diff
@@ -302,9 +304,16 @@ public Real(Real r)
...
 
-#region Interfaces
+		#region Interfaces
```
Comment (Lathreas) [inline engine/Real.cs:316]:
```suggestion
#region Interfaces
```
(These are just to keep things style-consistent with the rest of the file.)

**Review by Lathreas — APPROVED** [review]
Other than a small style thing, this looks good to me! It can be squash merged.

File: engine/Real.cs (hunk 2 above)
Comment (AdamNorberg) [reply engine/Real.cs]: Done. I kinda forgot the "commit suggestion" button exists, I should have just used it...

**Pull request merged**